src/ng/taskTrackerFactory.js

```diff
@@ -0,0 +1,122 @@
+'use strict';
+
+/**
+ * ! This is a private undocumented service !
```

Comment:
LOL - I can see the documentation below! :-P

Reply:
:grin: (I copied it from the docs of another private "undocumented" service 😛)